cdap-ui/app/cdap/components/EntityListView/NoEntitiesMessage/__tests__/NoEntitiesMessage.test.js

```diff
@@ -27,6 +27,9 @@ let noEntitiesMessage;
 let filtersAreApplied = jest.fn();
 
 describe('NoEntitiesMessage Unit tests - ', () => {
+  beforeEach(() => {
+    filtersAreApplied = jest.fn();
```

Comment:
I think this is unnecessary as we already have the function expression on line 27. Let me know if I'm missing something.